ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -126,6 +126,10 @@ literals:
 - outside_air_relative_humidity_sensor
 - outside_air_temperature_sensor
 - outside_air_wetbulb_temperature_sensor
+- outside_air_drybulb_temperature_sensor
```

Comment:
unless drybulb is really needed, it shouldnt be defined -- its expected that its a drybulb temp, unless wetbulb is used. if really needed, define it.

Reply:
Ohkay works